packages/gatsby-recipes/recipes/wordpress.mdx

```diff
@@ -0,0 +1,108 @@
+# Setup Gatsby with WordPress REST API 
+
+This recipes setups and configures gatsby-source-wordpress and starter page templates for a WordPress-powered Gatsby site.
+
+---
+
+Install necessary NPM packages
+
+<NPMPackage name="gatsby-source-wordpress" />
+
+---
+
+Install `gatsby-source-wordpress` to pull the WordPress data into Gatsby. Note, once this recipe finishes, you will need to update the config to point to your actual WordPress URL
+
+
+<GatsbyPlugin name="gatsby-source-wordpress" options={{
+  baseUrl: `cms.learnjam.org`,
+  protocol: `https`,
+  hostingWPCOM: false,
+  useACF: true,
+  includedRoutes: [
+      "**/posts",
+      "**/pages",
+      "**/users",
+      "**/categories",
+      "**/tags",
+  ]
+}} />
+
+---
+
+Create a local Gatsby plugin to create pages for the WordPress data
```

Comment:
Is this the best mechanism for this? I'd imagine editing gatsby-node.js is likely a bit easier and cleaner?

Reply:
Ah -- the challenges of editing a file that could/likely does exist. Scratch this -- makes sense!

Reply:
@DSchau This was requested by @KyleAMathews as a way to not directly overwrite the existing gatsby-node in case the person that uses this already has some custom code in their gatsby-node.

Reply:
unfortunately it is — a Gatsby Recipe can't automatically edit gatsby-node.js files — this is a strong motivation for putting route creation into the page component so this is simpler.